packages/runtime/src/services/debugger/type.ts

```diff
@@ -6,5 +6,5 @@ export type DebugLog = {
 };
 
 export type DebuggerHandler = {
-  onDebug: (currentState: Record<string, any>, debugLog: DebugLog) => void;
+  onDebug: (debugLog: DebugLog, currentState: Record<string, any>) => void;
```

Comment:
Why change the params order? Does it matters?

Reply:
The debuglog functions provided in the previous implementation were not aligned with onDebug, which resulted in the need to manually reorder them when using them   
before reordering
```ts
import { initSunmaoUI,printDebugInfo } from '@sunmao-ui/runtime';
  const ui = initSunmaoUI({
    // ...
    debugHandler:{
      onDebug:(state,log)=>{
        printDebugInfo(log,state);
      }
    }
  });
```
 after reordering
```ts
import { initSunmaoUI,printDebugInfo } from '@sunmao-ui/runtime';
  const ui = initSunmaoUI({
    // ...
    debugHandler:{
      onDebug: printDebugInfo
    }
  });
```
I think the impact is minimal, as this version is not yet released